content/millennium/r4/encounters/encounter.md

```diff
@@ -257,7 +257,7 @@ The common [errors] and [OperationOutcomes] may be returned.
 [`CodeableConcept`]: http://hl7.org/fhir/r4/datatypes.html#CodeableConcept
 [`Money`]: http://hl7.org/fhir/r4/datatypes.html#Money
 [`number`]: https://hl7.org/fhir/r4/search.html#number
-[`reference`]: http://hl7.org/fhir/r4/references.html#Reference
+[`reference`]: https://hl7.org/fhir/r4/search.html#reference
```

Comment:
Just out of curiosity, is there any reason these links are http vs. https?

Reply:
we want to set them all to https and not use http. a lot of links in the repository were initially added as http